components/setup/DockerInstructions.tsx

```diff
@@ -112,8 +113,10 @@ docker run -e INSTALL_ID=${response.data.installId} asia.gcr.io/vibi-prod/dpu/dp
 	};
 
 	const handleSubmit = () => {
+		setIsRepoSubmitButtonDisabled(true)
```

Comment:
You have to enable this again at the end of this function. (For both, success and failures cases)